### ISSUE TITLE
Tags needed for v 16.x

### ISSUE THREAD

**Issue by l-arnold** [turnkeylinux/tracker#1710]
I note that Tags are in place for
Odoo 14.0 14.1 14.2 and 15.0

We should tag 16.0 and 16.1 before moving to 17.0

expect this is an issue for other apps.
(I would vote for branches, but seems this is the method)

**Comment by JedMeister** [comment]
Yes they should be tagged (that's my bad)! Although I'm not 100% that I'll get to it before we start pushing out v17.0.

I know it's nowhere as near as good as a tag, but it should be possible to guess by looking at the commit history.
